### PR TITLE
 add exception

### PR DESCRIPTION
**Pull request by pantianying**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

including:
IllegalFormatWidthException
IllegalFormatConversionException
DuplicateFormatFlagsException
MissingResourceException
ConcurrentModificationException
RejectedExecutionException
CompletionException
EmptyStackException
IllformedLocaleException
NoSuchElementException

```

### CODE REVIEW

**Comment by pantianying** [comment]
add exception

File: java_exception_test.go [hunk 1]
```diff
@@ -56,6 +56,16 @@ func TestException(t *testing.T) {
 	doTestException(t, "throw_IndexOutOfBoundsException", "IndexOutOfBoundsException")
 	doTestException(t, "throw_ArrayIndexOutOfBoundsException", "ArrayIndexOutOfBoundsException")
 	doTestException(t, "throw_StringIndexOutOfBoundsException", "StringIndexOutOfBoundsException")
+	doTestException(t, "throw_IllegalFormatWidthException", "1000")
+	doTestException(t, "throw_IllegalFormatConversionException", "7 != test.TestThrowable$TestEnum")
+	doTestException(t, "throw_DuplicateFormatFlagsException", "null;DuplicateFormatFlagsException")
```
Comment (wongoo) [inline java_exception_test.go]: how about change the message `null;DuplicateFormatFlagsException` to `DuplicateFormatFlagsException` when `detailMessage` is nil?

Comment (pantianying) [reply java_exception_test.go]: ok I will change it

Comment (pantianying) [reply java_exception_test.go]: ok ，I will change it

**Comment by wongoo** [comment]
@pantianying thanks for the hard work!

**Pull request merged**